platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java

```diff
@@ -58,8 +77,13 @@ public class AddressBookTestingToolState extends PartialMerkleLeaf implements Sw
 
     private static final Logger logger = LogManager.getLogger(AddressBookTestingToolState.class);
 
-    /** modify this value to change how updateStake behaves. */
-    private static final int STAKING_PROFILE = 1;
+    /** the suffix for the debug address book */
+    private static String DEBUG = "debug";
+
+    private final AddressBookTestingToolConfig testingToolConfig =
+            ConfigurationHolder.getConfigData(AddressBookTestingToolConfig.class);
+    /** the address book configuration */
+    private final AddressBookConfig addressBookConfig = ConfigurationHolder.getConfigData(AddressBookConfig.class);
```

Comment:
Is it possible to do this without the ConfigurationHolder? We are actively trying to deprecate static access to configuraiton.

It should be possible to get the configuration inside the init() method, if that doesn't work then let me know and we can brainstorm ways to improve things.

Reply:
the config is now retrieved through init.